tests/integration/test_debug.py

```diff
@@ -271,6 +275,28 @@ def test_agentless(monkeypatch):
     assert info.get("agent_url", "AGENTLESS")
 
 
+def test_custom_writer():
+    tracer = ddtrace.Tracer()
+
+    class CustomWriter(TraceWriter):
+        def recreate(self):
+            # type: () -> TraceWriter
+            return self
+
+        def stop(self, timeout=None):
+            # type: (Optional[float]) -> None
+            pass
+
+        def write(self, spans=None):
+            # type: (Optional[List[Span]]) -> None
+            pass
+
+    tracer.writer = CustomWriter()
+    info = debug.collect(tracer)
+
+    assert info.get("agent_url", "CUSTOM")
```

Comment:
Should this be:
```suggestion
    assert info.get("agent_url", None) == "CUSTOM"
```

Reply:
lol I copied this check from the test above which also does the same thing 🤦 . Nice catch!